JSDemos/Demos/Calendar/Overview/Angular/app/app.component.html

```diff
@@ -70,6 +70,7 @@
       <span>First day of week</span>
       <dx-select-box
         [dataSource]="weekDays"
+        [inputAttr]="{ 'aria-label': 'Day' }"
```

Comment:
```suggestion
        [inputAttr]="{ 'aria-label': 'First Day of Week' }"
```
Please, update is all approcahes